### ISSUE TITLE
InitialHandler - bad packet ID

### ISSUE THREAD

**Issue by DanielAlves123** [HexagonMC/BungeeCord#274]
### What Hexacord version are you using? Post the output of /bungee
latest

### Are you using Bungee plugins? If yes, please list all of them + versionnumber
no

### How can we reproduce the error/bug? 
My console is spammed in these errors, probably some ddos attack on proxy
[WARNING] [/x:x] <-> InitialHandler - bad packet ID, are mods in use!? Did not read all bytes from packet class net.md_5.bungee.protocol.packet.Handshake 0 Protocol HANDSHAKE Direction TO_SERVER

### Is an error message (bungee/spigot/client)? If yes, use pastebin.com and share the link.
My console is spammed in these errors, probably some ddos attack on proxy
[WARNING] [/x:x] <-> InitialHandler - bad packet ID, are mods in use!? Did not read all bytes from packet class net.md_5.bungee.protocol.packet.Handshake 0 Protocol HANDSHAKE Direction TO_SERVER

### In addidtion to that, please explain your issue as detailed as possible.
My console is spammed in these errors, probably some ddos attack on proxy
[WARNING] [/x:x] <-> InitialHandler - bad packet ID, are mods in use!? Did not read all bytes from packet class net.md_5.bungee.protocol.packet.Handshake 0 Protocol HANDSHAKE Direction TO_SERVER

**Comment by TheEasyPeasy** [comment]
It's bungeecord exploit if you want to fix that contact with me on discord: TheEasyPeasy#5674

**Comment by spannerman79** [comment]
No, if there is a fix it needs to be fixed upstream. Not "in private via discord/etc"
